src/lib/txhandler.ts

```diff
@@ -115,7 +120,8 @@ export class SologenicTxHandler extends EventEmitter {
    */
   constructor(
     rippleApiOptions: SologenicTypes.RippleAPIOptions,
-    sologenicOptions: SologenicTypes.TransactionHandlerOptions
+    sologenicOptions: SologenicTypes.TransactionHandlerOptions,
```

Comment:
why don't we include the client options in SologenicOptions? I don't see why we need to have the third type. We can just easily add a property in sologenicOptions.